microdonuts/pom.xml

```diff
@@ -52,31 +52,33 @@
     <version>3.7.0</version>
   </dependency>
 
-  <!-- OpenTracing Tracer Implementations -->
   <dependency>
-    <groupId>com.lightstep.tracer</groupId>
-    <artifactId>lightstep-tracer-jre</artifactId>
-    <version>0.14.2</version>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-opentracing-shim</artifactId>
+    <version>0.7.0-20200727.220734-33</version>
```

Comment:
Is this version still accessible/download-able? Else, we should probably wait for the 0.7.0 release of the agent (and related components).

Reply:
yes, it's better to wait for 0.7.0.

Reply:
updated version to 0.7.0